CMakeLists.txt

```diff
@@ -1,28 +1,49 @@
 cmake_minimum_required(VERSION 3.1)
 
-project(opus LANGUAGES C)
+include(opus_functions.cmake)
+
+get_library_version(OPUS_LIBRARY_VERSION OPUS_LIBRARY_VERSION_MAJOR)
+message(STATUS "Opus library version: ${OPUS_LIBRARY_VERSION}")
+
+get_package_version(PACKAGE_VERSION)
+message(STATUS "Opus package version: ${PACKAGE_VERSION}")
+
+string(REGEX REPLACE "^v([0-9]+.[0-9]+\\.?([0-9]+)?).*" "\\1" PROJECT_VERSION ${PACKAGE_VERSION})
+
+project(Opus LANGUAGES C VERSION ${PROJECT_VERSION})
```

Comment:
Hmm, I think project() command clears a lot of stuff, so it makes sense to put it first right after `cmake_minimum_required()` no?

Reply:
This is ugly,  but allows `project` command to set up `PROJECT_VERSION_MINOR` `PROJECT_VERSION_MINOR` and other versions variables.

Acording to documentation, `cmake_minimum_required` must be the first command to set policies properly. `project` command is not required to be next command.

Reply:
Ok.